01-wasmcloud_basics/wash_intro/claims.md

```diff
@@ -0,0 +1,47 @@
+## wash claims
+
+The `wash claims` subcommand is used for generating and managing JWTs for wasmcloud actors.
```

Comment:
should probably let people know what a JWT is.. Maybe replace that with:

...generating and managing [JSON Web Tokens](https://jwt.io) (JWT)...